sdks/python/src/lilypad/_internal/otel/openai.py

```diff
@@ -274,7 +274,7 @@ def _process_chunk(
 
 
 def instrument_openai(
-    client: OpenAIClient,
+    client: _OpenAIClient,
```

Comment:
Isn't it better to use public rather than private for the argument type of public functions?